docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,311 @@
+never assume the identity or access." Before the Zero Trust security
+models, IT infrastructures were set up as a series of security
+perimeters. Think of as a castle with walls and moats. The castle would
+have a set number of entry points with guards. Once passed the guards
+and inside the castle, any visitors were trusted and had access to the
+castle. In the real world, passing the guards is analogous to
+authenticating with a machine or, at worst, connect the office network
+via WiFi or ethernet cable.
+
+Zero Trust does away with the concept of having a central castle that
+assumes anyone inside the castle is trusted. It assumes that the castle
+has already been breached. That is to say, in the real world, we expect
+attackers to already be inside the network and for it to be a hostile
+environment. Any resources inside the network should be treated as
+publicly available on the internet and must be defended. To accomplish
+this defense, a series of Zero Trust pillars are defined:
+
+- Never Trust, Verify - the virtue of a connection should not grant
+  access
+- Authenticate Before Connect - authentication should happen before
+  resources are connected to or requested
+- Least Privileged Access - access should only grant connectivity to the
+  minimum number of resources
+
+Those pillars, while short and vague, give rise to a challenge.
+Implementing those pillars is not a simple tweak to existing
+infrastructure. It is a series of challenges, and the first challenge
+will be how we move to a system that has the technology in place to
+verify connections? The answer to that is what this series calls
+"bootstrapping trust."
+
+### Ziti & Zero Trust
+
+In a Zero Trust model, there needs to exist mechanisms to verify
+identities such that trust can be granted. Zero Trust does not mean
+there is no trust. Zero Trust means that trust is given only after
+verification. Even then, that trust is limited to accessing the minimum
+network resources necessary. To accomplish this, we need a network that
+can force all connections through the following process.
+
+1. Authenticate
+2. Request Access To A Resource
+3. Connect To The Requested Resource
+
+To help make Zero Trust and Bootstrapping trust a bit clearer, it helps
+to have a concrete system to use an example. It just so happens that the
+Ziti software system makes a great example! Let us examine Ziti.
+
+In Ziti, all of the above steps require interacting with a Ziti
+Controller. The Ziti Controller manages the Ziti overlay network. It
+maintains a list of known network services, SDK clients, routers,
+enrollments, policies, and much more! All of these pieces working
+together to create a Ziti Network. A Ziti Network is an overlay network
+\- meaning it creates a virtual network on top of some other network
+referred to as the underlay network.
+
+In the Ziti Network, all network resources are modeled as services in
+the Ziti Controller. All services on a Ziti Network are only accessible
```

Comment:
"All services on a Ziti Network are only accessible via the Ziti Network." - i get what you mean but as written it's not factual, right?

Reply:
I had leavening terms (i.e. should) in there but removed them.

Reply:
Re-added "should....for maximum effect"